lib/services/ios-project-service.ts

```diff
@@ -357,6 +357,10 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 		return this.$fs.exists(path.join(projectRoot, this.$projectData.projectName, constants.APP_FOLDER_NAME));
 	}
 
+	public isPlatformBuilt(projectRoot: string): IFuture<boolean> {
+		return this.$fs.exists(path.join(projectRoot, "build"));
```

Comment:
Is this enough? Shouldn't you check for existing file with extensions `.ipa` or directory with name `*.app` inside this directory? What will happen in case it is empty?